### PR TITLE
Improved GNB model accuracy to 90%

### PR DESCRIPTION
**Pull request by khimabaja**
Introduced advanced feature extraction methods to improve accuracy to 90 percent. This commit contains new, improved implementation in a Jupyter notebook, along with the model as a .pkl file.

### CODE REVIEW

**Review by Avanznow — COMMENTED** [review]
Code approval process -- 3 Steps review:
(1) Yes; (2) Maybe; (3) No; > {Number of files, and total lines changed:add/delete lines} [Reasoning]
Example:

Feedback >> (2) > {4: +153/-20} [ Code needs to explain what it does; the need or issue it is helping to solve or improve; In which directory is it being placed; How it is used - add a README.md file is applicable; note: for naming files, use low case letters/underscore, e.g. rename_file.py; ]

**Review by Avanznow — CHANGES_REQUESTED** [review]
Feedback >> (2) > {2: +1762/-0} [ Following changes have to be solved. See in the sequence. ]

Please, use the same standard format and file naming according to the file purpose - for example:
trained_classifier_dl_jax.pkl
deep_learning_jax.ipynb
For example here:
https://github.com/Avanznow/Tellekynezix/tree/main/prediction-gaussiannb/jax

For the file you are submitting - it should be something like:
ref: https://github.com/Avanznow/Tellekynezix/tree/main/prediction_deep_learning/JAX
trained_classifier_gnb_jax.pkl
gaussianNB_train_jax.ipynb

Also, correct the file name: 
FROM - make sure to remove the "blob"
https://github.com/Avanznow/Tellekynezix/blob/main/prediction-gaussiannb/jax/GaussianNbJAX.md
https://github.com/3C-SCSU/Avatar/blob/main/prediction-gaussiannb/jax/README.md
See for example:
https://github.com/Avanznow/Tellekynezix/tree/main/prediction-gaussiannb/pytorch - no blob!

**Comment by khimabaja** [comment]
I have updated file names with requested changes. I am unsure about the blob files. The only ones present in this PR are the .pkl and .ipynb files
